index.rst

```diff
@@ -34,7 +30,12 @@ Tutorials List
 
 Quick Starts
 ~~~~~~~~~~~~
-`Associate metadata with Data <./dev_associate_metadata_quickstart/index.html>`__
+
+..
+    New Repositories will should be directly linked to the subproject path. For
+    example: `/projects/base-tutorial-repo/`
+
+* `Associate metadata with Data </projects/dev-associate-metadata-quickstart/>`_
```

Comment:
This link will only resolve on RTD for now. It can be replaced with a full link once you have a domain set up for your documentation.